app/views/bulkrax/entries/show.html.erb

```diff
@@ -0,0 +1,56 @@
+<div class="panel panel-default">
+  <div class="panel-body">
+
+    <p id="notice"><%= notice %></p>
+
+    <p>
+      <strong>Identifier:</strong>
+      <%= @entry.identifier %>
+    </p>
+
+    <p>
+      <strong>Entry ID:</strong>
+      <%= @entry.id %>
+    </p>
+
+    <p>
+      <strong>Type:</strong>
+      <%= @entry.factory_class %>
+    </p>
+
+    <p>
```

Comment:
for raw_metadata and parsed_metadata, could you format the hash like the importer ones ... I'd suggest checking whether the data is a hash first (as for some entries it could be a blob of text or something else).

So, 

```
if x.present? && x.is_a?(Hash)
  format it
else
  just print it
end
```